### DISCUSSION TITLE
Upcoming Community PRs to be merged

### DISCUSSION THREAD

**Discussion by gurpreet-wpf** [dotnet/wpf#6178] [Announcements]
Hi folks,

Below is the list of community PRs which we are planning to merge in the upcoming weeks:
1. #6044 
2. #5933 
3. #5922 
4. #5915 
5. #5814 
6. #5182 
7. #6060 
8. #6059 

Thanks!

**Comment by ThomasGoulet73** [comment]
@gurpreet-wpf Are there any news on this ? When could we expect those PRs to be merged ?

**Reply by gurpreet-wpf** [reply]
@ThomasGoulet73 All CTP PRs are merged except #5915

**Reply by ThomasGoulet73** [reply]
@gurpreet-wpf Thank you!
